src/effects/effectsmanager.h

```diff
@@ -21,6 +21,8 @@ class EffectChainManager;
 class EffectManifest;
 class EffectsBackend;
 
+#define MIXXX_NOEFFECT_STRING " --- "
```

Comment:
```suggestion
namespace {
const QString kNoEffectString = QStringLiteral(" --- ");
} // namespace
```

Reply:
:facepalm: tried that in the cpp before...not in the header

Reply:
Yes, anonymous namespace do not belong in header files.

Reply:
so, should I stick with the pevious MIXXX_NOEFFECT_STRING or what do you propose?

Reply:
Oh I did not notice that this is a header file. 
We should avoid type less macros, the pollute the global name space. 
It can become a static member variable of the effectsmanager. This should be than initialized in the CPP file.

Reply:
like: https://github.com/mixxxdj/mixxx/blob/3085bbe95107fd789a8a966dfeb341cfe74cfad8/src/control/controlbehavior.cpp#L290

Reply:
yes, I already decided to go with that, like https://github.com/mixxxdj/mixxx/blob/2.3/src/soundio/soundmanagerconfig.cpp#L14